subsys/bluetooth/rpc/client/bt_rpc_internal_client.c

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
Do we need separated source file for this function ?

Reply:
A separate module was created to separate the functions related to internal testing from rest.
Currently one feature is added, but others may need to be added in the future.